docs/development.md

```diff
@@ -261,12 +261,11 @@ to build an image and a deployment config.
 
 ### Automated Deployment
 
-If you just want to have this automated, then jump to the [automated deployment
-steps in the userguide](userguide.md#automated-deployment) but be sure to
-specify your own image such as:
+If you just want to have this automated, then run the following command
+specifying your own image:
 
 ```bash
-./scripts/deploy-federation-latest.sh <containerregistry>/<username>/federation-v2:test cluster2
+./scripts/deploy-federation.sh <containerregistry>/<username>/federation-v2:test cluster2
```

Comment:
@font I think using `cluster2` here is not good, also we may need to clarify that here `cluster2` need to be same as the joining cluster context.

How about update here as follows:

```bash
./scripts/deploy-federation.sh <containerregistry>/<username>/federation-v2:test <join-cluster1> <join-cluster2>
```

**NOTE**: You can list multiple joining cluster names in above command. Please make sure the joining cluster name is same as joining cluster context.

Reply:
I think adding a note about the joining cluster context matching the cluster name is good. I will leave it as `cluster2` because it follows with the rest of the guide https://github.com/font/federation-v2/blob/840eb7709102c6f8bdede7eb0f0f35b0fc55c1cd/docs/development.md#setup-clusters-deploy-the-cluster-registry-and-federation-v2-control-plane but will make a note about specifying any number of clusters.